vic/extensions/rout_rvic/src/rout_mpi_scatter.c

```diff
@@ -0,0 +1,80 @@
+/******************************************************************************
+ * @section DESCRIPTION
+ *
+ * Allocate memory for Routing structures.
+ *
+ * @section LICENSE
+ *
+ * The Variable Infiltration Capacity (VIC) macroscale hydrological model
+ * Copyright (C) 2014 The Land Surface Hydrology Group, Department of Civil
+ * and Environmental Engineering, University of Washington.
+ *
+ * The VIC model is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *****************************************************************************/
+
+#include <vic_def.h>
+#include <vic_run.h>
+#include <vic_driver_image.h>
+#include <rout.h>
+
+/******************************************************************************
+ * @brief   Scatter double precision variable
+ * @details values from master node are scattered to the local nodes
+ *****************************************************************************/
+void get_scatter_var_double(double *dvar, double *var_local) {
```

Comment:
don't we have this function in `image_shared`? If not, let's just put these in `vic_mpi_support.c`.